test/boost/canonical_mutation_test.cc

```diff
@@ -29,7 +29,7 @@ SEASTAR_TEST_CASE(test_conversion_back_and_forth) {
 
 SEASTAR_THREAD_TEST_CASE(test_conversion_back_and_forth_gently) {
     for_each_mutation([] (const mutation& m) {
-        canonical_mutation cm(m);
+        auto cm = make_canonical_mutation_gently(m).get();
```

Comment:
Again need to check that the ungentle version matches the gentle version, without relying on to_mutation_gently() that is being tested here. You're testing two independent things in one test.

Reply:
will fix in next version

Reply:
fixed in v5